README.md

```diff
@@ -20,26 +20,33 @@ Minimal and straightforward when doing queries
 ```python
 from nmi_mysql import nmi_mysql
 ```
-- Initialization: Accepts two parameters, first being the config object and the second specifying if autoconnect to db is enabled. If set to false, call `con.connect()` 
+
+- Initialization: Accepts two parameters, first being the config object and the second specifying the maximum number of connections in the pool (default is 20)
```

Comment:
We will but the example you gave takes 1 parameter, no? The max connection should be defined in the config object?

db = nmi_mysql.DB(conf)